torch/_tensor_docs.py

```diff
@@ -1784,11 +1784,11 @@ def add_docstr_all(method, docstr):
 
 Args:
     dim (int): dimension along which to index
-    index (IntTensor or LongTensor): indices of :attr:`tensor` to select from
-    tensor (Tensor): the tensor containing values to add
+    index (IntTensor or LongTensor): indices of :attr:`input` to select from
```

Comment:
While we're here the language "IntTensor" and "LongTensor" is deprecated. Index is just a tensor. The requirement that it have the long (int64) or int (int32) dtype can be noted independently of the Python  type

Reply:
Thanks, @mruberry for your suggestion. I think this can be added in the same line after " to select from" (added in the recent commit). Does it sound good to you?